otter/worker/_rcv3.py

```diff
@@ -27,6 +28,11 @@ def _generic_rcv3_request(step_class, request_bag, lb_id, server_id):
         request has no body.
     """
     effect = step_class(lb_node_pairs=[(lb_id, server_id)])._bare_effect()
+
+    svc_req = effect.intent
```

Comment:
I think for deletes only, most of the 409s are ok?  If the node isn't a member, that's fine.  If the load balancer pool doesn't exist anymore, that's fine.  If the load balancer pool isn't active... uh, I guess the RCv3 team said they'd deal with errors like that, right?